packages/storybook/src/generators/configuration/lib/util-functions.ts

```diff
@@ -312,6 +415,16 @@ export function addStorybookToNamedInputs(tree: Tree) {
       ) {
         nxJson.namedInputs.production.push('!{projectRoot}/.storybook/**/*');
       }
+
+      if (
+        !nxJson.namedInputs.production.includes(
+          '!{projectRoot}/tsconfig.storybook.json'
+        )
+      ) {
+        nxJson.namedInputs.production.push(
+          '!{projectRoot}/tsconfig.storybook.json'
```

Comment:
I think this should be {projectRoot}/tsconfig.storybook.json.

We'd want to force a rebuild if the tsconfig changes. As it stands right now, changes could be made to it without it impacting the cache, right?

Reply:
I think we would not want to force a rebuild of the app, but a rebuild of storybook, so I think it's ok?

Reply:
In the same sense that we ignore tsconfig.spec.json

Reply:
oh yes, i see this is being added to the production group, sorry, my bad!